Formula/p/python@3.11.rb

```diff
@@ -66,26 +67,6 @@ class PythonAT311 < Formula
   link_overwrite "Frameworks/Python.framework/Versions/Current"
 
   # Always update to latest release
-  resource "flit-core" do
-    url "https://files.pythonhosted.org/packages/c4/e6/c1ac50fe3eebb38a155155711e6e864e254ce4b6e17fe2429b4c4d5b9e80/flit_core-3.9.0.tar.gz"
-    sha256 "72ad266176c4a3fcfab5f2930d76896059851240570ce9a98733b658cb786eba"
-  end
-
-  resource "setuptools" do
-    url "https://files.pythonhosted.org/packages/ef/cc/93f7213b2ab5ed383f98ce8020e632ef256b406b8569606c3f160ed8e1c9/setuptools-68.2.2.tar.gz"
-    sha256 "4ac1475276d2f1c48684874089fefcd83bd7162ddaafb81fac866ba0db282a87"
-  end
-
-  resource "pip" do
-    url "https://files.pythonhosted.org/packages/ba/19/e63fb4e0d20e48bd2167bb7e857abc0e21679e24805ba921a224df8977c0/pip-23.2.1.tar.gz"
-    sha256 "fb0bd5435b3200c602b5bf61d2d43c2f13c02e29c1707567ae7fbc514eb9faf2"
-  end
-
-  resource "wheel" do
-    url "https://files.pythonhosted.org/packages/a4/99/78c4f3bd50619d772168bec6a0f34379b02c19c9cced0ed833ecd021fd0d/wheel-0.41.2.tar.gz"
-    sha256 "0c5ac5ff2afb79ac23ab82bab027a0be7b5dbcf2e54dc50efe4bf507de1f7985"
-  end
-
```

Comment:
looks like a bug? cc @woodruffw @alex

Reply:
Yeah, I'm not sure why this started happening now. My first guess is that it's because these deps need to be listed in the PyPI formula mapping; I can look into that in a moment.

Reply:
that is fine, just notify you about the issue :)

Reply:
@woodruffw there is come context in this thread:
- https://github.com/Homebrew/homebrew-core/pull/127641
- https://github.com/pypa/wheel/pull/501